src/geometry/GeoParams.cc

```diff
@@ -102,7 +105,7 @@ const std::string& GeoParams::id_to_label(VolumeId vol_id) const
 {
     CELER_EXPECT(vol_id.get() < num_volumes_);
     const auto* vol
-        = vecgeom::GeoManager::Instance().FindPlacedVolume(vol_id.get());
+        = vecgeom::GeoManager::Instance().FindLogicalVolume(vol_id.get());
     CELER_ASSERT(vol);
     return vol->GetLabel();
```

Comment:
The `vecgeom::LogicalVolume` has a `std::string GetLabel()` and a `const char* GetName()`. Do you know if there is any particular reason for having two different functions, with different names, that return essentially the same information? Especially if the terms `label` and `name` could have different meanings in vecgeom -- which I don't know if it is the case, but it is worth pointing that out.

Reply:
These different terms, label and name, are basically interchangeable in VecGeom.  I think the original was label, and then the name was added for easy compatibility with Geant4.